app/models/source.rb

```diff
@@ -25,7 +25,14 @@ def my
 
     # @return [Template] The default template.
     def default_template
-      default = Template.new(Rails.root.join("example_templates", "torque").to_s, Source.new("Examples", Rails.root.join("example_templates").to_s))
+      # TODO Provide an LSF script and a default in case adapter is something else
+      adapter = OODClusters.find{|x| !x.job_config[:adapter].nil? }.job_config[:adapter] || 'torque'
+      if adapter == 'torque'
+        default = Template.new(Rails.root.join("example_templates", "torque").to_s, Source.new("Examples", Rails.root.join("example_templates").to_s))
+      elsif adapter == 'slurm'
+        default = Template.new(Rails.root.join("example_templates", "slurm").to_s, Source.new("Examples", Rails.root.join("example_templates").to_s))
+      end
+
```

Comment:
Instead of adding torque and slurm default templates, how about we just replace the original template with a script that doesn't have any headers. We could comment out example headers for PBS and for SLURM and LSF etc. and ask the user to modify with their own headers.